main.py

```diff
@@ -294,38 +297,70 @@ async def command_list(message: Message):
 
     # Scrape all tracks in the target channel and list them
     channel_media_attachments = await scrape_channel_media(target_channel)
-
+    # title of !list embed
     embed_title = "❤️‍🔥 AIGHT. IT'S BUSTY TIME ❤️‍🔥"
-    embed_content = "**Track Listing**\n"
+    embed_description_prefix = "**Track Listing**\n"
+
+    # stack of embed descriptions to circumvent the 4096 character embed limit
+    embed_description_stack = []
+    embed_description_current = ""
 
     for index, (
         submit_message,
         attachment,
         local_filepath,
     ) in enumerate(channel_media_attachments):
         list_format = "**{0}.** {1}: [{2}]({3}) [`↲jump`]({4})\n"
-        embed_content += list_format.format(
+        list_entry = list_format.format(
             index + 1,
             submit_message.author.mention,
             format_filename(attachment.filename),
             attachment.url,
             submit_message.jump_url,
         )
+        if (
+            len(embed_description_prefix)
+            + len(embed_description_current)
+            + len(list_entry)
+            > embed_description_limit
+        ):
+            # if adding a new list entry would go over, push our current list entries to an embed
+            embed_description_stack.append(embed_description_current)
+            # start a new embed
+            embed_description_current = list_entry
+        else:
+            embed_description_current += list_entry
+
+    # add the leftover part to a new embed (it it exists)
+    if len(embed_description_current) > 0:
+        embed_description_stack.append(embed_description_current)
 
-    # Send the list message
-    embed = discord.Embed(title=embed_title, description=embed_content, color=0xDD2E44)
-    list_message = await message.channel.send(embed=embed)
+    # iterate through each embed description, send and stack messages
+    message_stack = []
+    if len(embed_description_stack) == 0:
+        await message.channel.send("There aint any songs there.")
+        return
 
-    # Only pin the message if the command is called from the target channel
+    for embed_description in embed_description_stack:
+        embed = discord.Embed(
+            title=embed_title,
+            description=embed_description_prefix + embed_description,
```

Comment:
Is it possible to only include the title and prefix for the first embed?

Perhaps with:

```python
for entry_index, embed_description in enumerate(embed_description_stack):
    if entry_index == 0:
        ...
    else:
        ...
```